### PR TITLE
[zh-cn]Reference files to sync task 3

### PR DESCRIPTION
**Pull request by asa3311**
Reference files to sync task 3

Related to task-3 of issue https://github.com/kubernetes/website/issues/39987.

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ NetworkPolicySpec 定义特定 NetworkPolicy 所需的所有信息.
       IPBlock defines policy on a particular IPBlock. If this field is set then neither of the other fields can be.
 
       <a name="IPBlock"></a>
-      *IPBlock describes a particular CIDR (Ex. "192.168.1.1/24","2001:db9::/64") that is allowed to the pods matched by a NetworkPolicySpec's podSelector. The except entry describes CIDRs that should not be included within this rule.*
+      *IPBlock describes a particular CIDR (Ex. "192.168.1.0/24","2001:db8::/64") that is allowed to the pods matched by a NetworkPolicySpec's podSelector. The except entry describes CIDRs that should not be included within this rule.*
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md:139]: Please change the zh text accordingly

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md [hunk 2]
```diff
@@ -151,11 +151,11 @@ NetworkPolicySpec 定义特定 NetworkPolicy 所需的所有信息.
       <!--
       - **ingress.from.ipBlock.cidr** (string), required
 
-        CIDR is a string representing the IP Block Valid examples are "192.168.1.1/24" or "2001:db9::/64"
+        CIDR is a string representing the IP Block Valid examples are "192.168.1.0/24" or "2001:db8::/64"
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md:154]: Please change the zh text accordingly

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md [hunk 3]
```diff
@@ -151,11 +151,11 @@ NetworkPolicySpec 定义特定 NetworkPolicy 所需的所有信息.
...
 
       - **ingress.from.ipBlock.except** ([]string)
 
-        Except is a slice of CIDRs that should not be included within an IP Block Valid examples are "192.168.1.1/24" or "2001:db9::/64" Except values will be rejected if they are outside the CIDR range
+        Except is a slice of CIDRs that should not be included within an IP Block Valid examples are "192.168.1.0/24" or "2001:db8::/64" Except values will be rejected if they are outside the CIDR range
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md:158]: Please change the zh text accordingly

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md [hunk 4]
```diff
@@ -302,7 +302,7 @@ NetworkPolicySpec 定义特定 NetworkPolicy 所需的所有信息.
       IPBlock defines policy on a particular IPBlock. If this field is set then neither of the other fields can be.
 
       <a name="IPBlock"></a>
-      *IPBlock describes a particular CIDR (Ex. "192.168.1.1/24","2001:db9::/64") that is allowed to the pods matched by a NetworkPolicySpec's podSelector. The except entry describes CIDRs that should not be included within this rule.*
+      *IPBlock describes a particular CIDR (Ex. "192.168.1.0/24","2001:db8::/64") that is allowed to the pods matched by a NetworkPolicySpec's podSelector. The except entry describes CIDRs that should not be included within this rule.*
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md:305]: Please change the zh text accordingly

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md [hunk 5]
```diff
@@ -317,11 +317,11 @@ NetworkPolicySpec 定义特定 NetworkPolicy 所需的所有信息.
       <!--
       - **egress.to.ipBlock.cidr** (string), required
 
-        CIDR is a string representing the IP Block Valid examples are "192.168.1.1/24" or "2001:db9::/64"
+        CIDR is a string representing the IP Block Valid examples are "192.168.1.0/24" or "2001:db8::/64"
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md:320]: Please change the zh text accordingly

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md [hunk 6]
```diff
@@ -317,11 +317,11 @@ NetworkPolicySpec 定义特定 NetworkPolicy 所需的所有信息.
...
 
       - **egress.to.ipBlock.except** ([]string)
 
-        Except is a slice of CIDRs that should not be included within an IP Block Valid examples are "192.168.1.1/24" or "2001:db9::/64" Except values will be rejected if they are outside the CIDR range
+        Except is a slice of CIDRs that should not be included within an IP Block Valid examples are "192.168.1.0/24" or "2001:db8::/64" Except values will be rejected if they are outside the CIDR range
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md:324]: Please change the zh text accordingly

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 7]
```diff
@@ -363,33 +363,35 @@ ServiceSpec 描述用户在服务上创建的属性。
...
   -->
-  externalTrafficPolicy 表示此 Service 是否希望将外部流量路由到节点本地或集群范围的 Endpoint。
-  字段值 “Local” 保留客户端 IP 并可避免 LoadBalancer 和 Nodeport 类型 Service 的第二跳，但存在潜在流量传播不平衡的风险。
-  字段值 “Cluster” 则会掩盖客户端源 IP，可能会导致第二次跳转到另一个节点，但整体流量负载分布较好。
+  externalTrafficPolicy 描述了节点如何分发它们在 Service 的“外部访问”地址（NodePort、ExternalIP 和 LoadBalancer IP）接收到的服务流量。
+  如果设置为“Local”，代理将以一种假设外部负载均衡器将负责在节点之间服务流量负载均衡，因此每个节点将仅向服务的节点本地端点传递流量，而不会伪装客户端源 IP。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:369]:
```suggestion
  如果设置为 “Local”，代理将以一种假设外部负载均衡器将负责在节点之间服务流量负载均衡，
  因此每个节点将仅向服务的节点本地端点传递流量，而不会伪装客户端源 IP。
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 8]
```diff
@@ -363,33 +363,35 @@ ServiceSpec 描述用户在服务上创建的属性。
...
+ （将丢弃错误发送到没有端点的节点的流量。）
+  “Cluster”默认值使用负载均衡路由到所有端点的策略（可能会根据拓扑和其他特性进行修改）。
+  请注意，从集群内部发送到 External IP 或 LoadBalancer IP 的流量始终具有“Cluster”语义，但是从集群内部发送到 NodePort 的客户端需要在选择节点时考虑流量路由策略。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:372]:
```suggestion
  请注意，从集群内部发送到 External IP 或 LoadBalancer IP 的流量始终具有 “Cluster” 语义，
  但是从集群内部发送到 NodePort 的客户端需要在选择节点时考虑流量路由策略。
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 9]
```diff
@@ -363,33 +363,35 @@ ServiceSpec 描述用户在服务上创建的属性。
...
 
   <!-- 
-  InternalTrafficPolicy specifies if the cluster internal traffic should be routed to all endpoints or node-local endpoints only. "Cluster" routes internal traffic to a Service to all endpoints. "Local" routes traffic to node-local endpoints only, traffic is dropped if no node-local endpoints are ready. The default value is "Cluster". 
+  InternalTrafficPolicy describes how nodes distribute service traffic they receive on the ClusterIP. If set to "Local", the proxy will assume that pods only want to talk to endpoints of the service on the same node as the pod, dropping the traffic if there are no local endpoints. The default value, "Cluster", uses the standard behavior of routing to all endpoints evenly (possibly modified by topology and other features).
   -->
-  internalTrafficPolicy 指定是将集群内部流量路由到所有端点还是仅路由到节点本地的端点。
-  字段值 “Cluster” 将 Service 的内部流量路由到所有端点。
-  字段值 ”Local” 意味着仅将流量路由到节点本地的端点；如果节点本地端点未准备好，则丢弃流量。
-  默认值为 “Cluster”。
+  InternalTrafficPolicy描述节点如何分发它们在ClusterIP上接收到的服务流量。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:379]:
```suggestion
  internalTrafficPolicy 描述节点如何分发它们在 ClusterIP 上接收到的服务流量。
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 10]
```diff
@@ -363,33 +363,35 @@ ServiceSpec 描述用户在服务上创建的属性。
...
+  如果设置为"Local"，代理将假定pod只想与在同一节点上的服务端点通信，如果没有本地端点，它将丢弃流量。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:380]:
```suggestion
  如果设置为 “Local”，代理将假定 Pod 只想与在同一节点上的服务端点通信，如果没有本地端点，它将丢弃流量。
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 11]
```diff
@@ -363,33 +363,35 @@ ServiceSpec 描述用户在服务上创建的属性。
...
+  "Cluster"默认将流量路由到所有端点（可能会根据拓扑和其他特性进行修改）。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:381]:
```suggestion
  “Cluster” 默认将流量路由到所有端点（可能会根据拓扑和其他特性进行修改）。
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 12]
```diff
@@ -363,33 +363,35 @@ ServiceSpec 描述用户在服务上创建的属性。
...
   -->
-  externalTrafficPolicy 表示此 Service 是否希望将外部流量路由到节点本地或集群范围的 Endpoint。
-  字段值 “Local” 保留客户端 IP 并可避免 LoadBalancer 和 Nodeport 类型 Service 的第二跳，但存在潜在流量传播不平衡的风险。
-  字段值 “Cluster” 则会掩盖客户端源 IP，可能会导致第二次跳转到另一个节点，但整体流量负载分布较好。
+  externalTrafficPolicy 描述了节点如何分发它们在 Service 的“外部访问”地址（NodePort、ExternalIP 和 LoadBalancer IP）接收到的服务流量。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:368]:
```suggestion
  externalTrafficPolicy 描述了节点如何分发它们在 Service 的“外部访问”地址
  （NodePort、ExternalIP 和 LoadBalancer IP）接收到的服务流量。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Pull request merged**